src/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -166,7 +166,9 @@ public partial class X509Certificate : System.IDisposable, System.Runtime.Serial
         public virtual byte[] Export(System.Security.Cryptography.X509Certificates.X509ContentType contentType, string password) { throw null; }
         protected static string FormatDate(System.DateTime date) { throw null; }
         public virtual byte[] GetCertHash() { throw null; }
+        public virtual byte[] GetCertHash(HashAlgorithmName hashAlgorithm) { throw null; }
         public virtual string GetCertHashString() { throw null; }
+        public virtual string GetCertHashString(HashAlgorithmName hashAlgorithm) { throw null; }
```

Comment:
Do we really need to do GetCertHashString? I would think of the *String APIs as mostly unnecessary.

Reply:
Just symmetry